applications/wg-easy/replicated/application.yaml

```diff
@@ -9,11 +9,12 @@ spec:
   allowRollback: true
   #additionalImages:
   #  - jenkins/jenkins:lts
-  additionalNamespaces:
-    - "cert-manager"
-    - "traefik"
-    - "wg-easy"
-    - "*"
+  # additionalNamespaces should be populated by the Task file
+  # additionalNamespaces:
+  #   - "cert-manager"
+  #   - "traefik"
+  #   - "wg-easy"
+  #   - "*"
```

Comment:
You can remove the u nused values and just keep the comment about auto-generation.